recipes/nvidia-gpu/manifests/1000-state-driver.yaml

```
@@ -40,17 +40,18 @@ data:
     #!/bin/bash -x
```

I was thinking that this script should use `set -e`  to fail on any error and not continue with unexpected (empty) values

---

but if the fist attempt fails, how can we make sure it moves to the second stage and not exits due the `set -e` ?

---

by making sure that no command ever "blindly" fails, eg:

```
set -e
dnf install invalid # bad, returns 1 ==> exits

dnf install invalid && SUCCEED=1 || SUCCEED=0 # good, returns 0 and sets $SUCCEED
if dnf install invalid; then ...; else ... ;fi # good as well
```